### PR TITLE
Fix styling issues

### PR DESCRIPTION
**Pull request by torztomasz**
Resolves L2B-586, L2B-545

This PR contains a lot of small UI fixes.

### CODE REVIEW

File: packages/frontend/src/components/header/DetailsHeader.stories.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ function Template({ projectId }: TemplateProps) {
       value: <StatWithChange stat="2.21" change="-10.23%" />,
     },
     {
-      title: '30d Transaction Count',
+      title: '30D ts count',
```
Comment (sz-piotr) [inline packages/frontend/src/components/header/DetailsHeader.stories.tsx:35]:
```suggestion
      title: '30D tx count',
```

File: packages/frontend/src/pages/scaling-projects/view/ProjectHeader.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ export function ProjectHeader(props: ProjectHeaderProps) {
         ),
     },
     {
-      title: '30d transaction count',
+      title: '30D ts count',
```
Comment (sz-piotr) [inline packages/frontend/src/pages/scaling-projects/view/ProjectHeader.tsx:50]:
```suggestion
      title: '30D tx count',
```

File: packages/frontend/src/components/HorizontalSeparator.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import cx from 'classnames'
+import React from 'react'
+export interface SeparatorProps {
+  className?: string
+}
+export function HorizontalSeparator(props: SeparatorProps) {
+  const { className } = props
```
Comment (sz-piotr) [inline packages/frontend/src/components/HorizontalSeparator.tsx:7]: We never use this convention anywhere. We do:
- either destructure in the function signature, e.g. `function Component({ a, b }: ComponentProps`
- or just use `props.a` and `props.b` directly

Comment (sz-piotr) [reply packages/frontend/src/components/HorizontalSeparator.tsx:7]: You can also update KnowledgeNuggetItem to follow this :) as I missed it during your previous PR

**Review by sz-piotr — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/HorizontalSeparator.tsx (hunk 3 above)
Comment (torztomasz) [reply packages/frontend/src/components/HorizontalSeparator.tsx:7]: Okay, I have to change my habits from previous projects 👍

**Review by sz-piotr — APPROVED** [review]

**Pull request merged**